packages/salesforcedx-vscode-core/src/commands/baseDeployRetrieve.ts

```diff
@@ -71,9 +83,7 @@ export abstract class DeployRetrieveExecutor<
       // getComponents uses ComponentSet from SDR which assigns
       // the default latest value to components.apiversion
 
-      const apiVersion = (await ConfigUtil.getConfigValue('apiVersion')) as
-        | string
-        | undefined;
+      const apiVersion = await getApiVersion();
```

Comment:
Ideally this will be relocated to a configUtil - something for a later refactor, because it was causing issues being moved here.

Reply:
Actually @klewis-sfdc can you leave that as a code comment?